### PR TITLE
README.md: Fix up the 'Start Developing instructions

### PR DESCRIPTION
**Pull request by stefwalter**
1. Most distributions need sudo to run kube and/or access docker
 2. Update URL in instructions
 3. Add admin user to default project that hello-pod.json ends up in

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -88,15 +88,17 @@ Once setup with a Go development environment and Docker, you can:
...
+        $ sudo make run
 
 3.  In another terminal window, switch to the directory and start an app:
 
         $ cd $GOPATH/src/github.com/openshift/origin
         $ export OPENSHIFTCONFIG=`pwd`/openshift.local.config/master/admin.kubeconfig 
         $ _output/local/go/bin/osc create -f examples/hello-openshift/hello-pod.json
+        $ _output/local/go/bin/osadm policy add-role-to-user admin admin -n default
 
-In your browser, go to [http://localhost:6061](http://localhost:6061) and you should see 'Welcome to OpenShift'.
```
Comment (liggitt) [inline README.md]: 6061 is the port the pod exposes

File: README.md [hunk 2]
```diff
@@ -88,15 +88,17 @@ Once setup with a Go development environment and Docker, you can:
...
+
+In your browser, go to [https://localhost:8443](https://localhost:8443) and you should see 'Welcome to OpenShift'. Use 'admin' as the user and 'admin' as the password.
```
Comment (liggitt) [inline README.md]: This is explaining how to access the port exposed by the running pod. You don't need to log in to do that

**Comment by stefwalter** [comment]
Fixed wording to be clear that you're accessing the running pod.

**Comment by mfojtik** [comment]
LGTM [merge]

**Pull request merged**